src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <>
+    <div className="main">
+      <Operand />
+      <div className="buttons">
+        <div className="left-section">
+          <div className="ac">AC</div>
+          <div className="plusOrMinus">+/-</div>
+          <div className="modelus">%</div>
+          <div className="seven">7</div>
+          <div className="eight">8</div>
+          <div className="nine">9</div>
+          <div className="four">4</div>
+          <div className="five">5</div>
+          <div className="six">6</div>
+          <div className="1">1</div>
+          <div className="two">2</div>
+          <div className="three">3</div>
+          <div className="zero">0</div>
+          <div className="dot">.</div>
```

Comment:
[OPTIONAL] Great work done by implementing by building this calculators , I would just suggest that you create a component for these operand instead of repeating the same thing and pass the value and the id as arguments inside a for loop to get DRY code

Reply:
Thanks for your feedback. I will consider your suggestion and will do as you suggested.